### PR TITLE
Llamafile - Gemma 2 Colab Notebook 

### PR DESCRIPTION
**Pull request by kinarr**
This demonstrates how to use Llamafile with Gemma 2

### CODE REVIEW

**Comment by windmaple** [comment]
Can you add an entry in the table of contents in README?

**Comment by kinarr** [comment]
> Can you add an entry in the table of contents in README?

Added it now and also removed the entry from the WISHLIST

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ You can find the Gemma models on GitHub, Hugging Face models, Kaggle, Google Clo
...
 | [RAG_PDF_Search_in_multiple_documents_on_Colab.ipynb](Gemma/RAG_PDF_Search_in_multiple_documents_on_Colab.ipynb)                                 | RAG PDF Search in multiple documents using Gemma 2 2B on Google Colab.                                                                                                                  |
 | [Using_Gemma_with_LangChain.ipynb](Gemma/Using_Gemma_with_LangChain.ipynb)                                                                       | Examples to demonstrate using Gemma with [LangChain](https://www.langchain.com/).                                                                                                       |
+| [Using_Gemma_with_Llamafile.ipynb](Gemma/Using_Gemma_with_Llamafile.ipynb)                                                                       | An example to demonstrate using Gemma with [Llamafile](https://github.com/Mozilla-Ocho/llamafile/).                                                                                     |
```
Comment (windmaple) [inline README.md]: Move this up to below Run_with_Ollma.ipynb. This position is for RAG.

**Review by windmaple — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kinarr) [reply README.md]: Done.

**Review by windmaple — APPROVED** [review]

**Pull request merged**